src/test/scala/com/cognite/sdk/scala/v1/FilesTest.scala

```diff
@@ -142,13 +143,18 @@ class FilesTest extends SdkTest with ReadBehaviours with WritableBehaviors {
   }
   it should "support upload" in {
     val file =
-      client.files.uploadWithName(
-        new java.io.File("./src/test/scala/com/cognite/sdk/scala/v1/uploadTest.txt"),
+      client.files.upload(
+        new io.FileInputStream("./src/test/scala/com/cognite/sdk/scala/v1/uploadTest.txt"),
         "uploadTest123.txt"
       )
 
     client.files.deleteById(file.id)
 
     assert(file.name == "uploadTest123.txt")
   }
+  it should "support download" in {
+    val out = new io.ByteArrayOutputStream()
+    client.files.download(FileDownloadId(3655755582387145L), out)
+    assert(out.toString() == "a")
```

Comment:
Shouldn't this be `This is a test file.`? Perhaps even better, check that it is byte-by-byte identical to `Files.readAllBytes(Paths.get("./src/test/scala/com/cognite/sdk/scala/v1/uploadTest.txt"))`, or convert both byte arrays to strings with UTF-8 encoding and compare the results.